### ISSUE TITLE
All text has no space between the words

### ISSUE THREAD

**Issue by Ruke805** [hamsolo474/VideoSubFinder_ocr_path#1]
Hi, I've tested your program...I edit the build subs.py to add my language to be recognized, and works...but all words recognized, was written with no space between it... 

"Something to be like this" ...was saved as "Somethingtobelikethis"

Could help me?

**Comment by hamsolo474** [comment]
Yeah, i know what causes that.
Tesseract tries to put spaces anywhere it doesn't see a letter, so I added
a line to remove spaces from Tesseract's output, which works for Chinese
which is what I was using it for, but obviously doesn't work for other
languages.

In the text fixer function inside the replaceMe variable, remove the space.
It's the second last entry on line 139

**Comment by theruleof4** [comment]
Its to edit build subs.py to fix the problem? Sorry, my knowledge about programming its very basic.

**Comment by hamsolo474** [comment]
Yeah it's very easy, open build_subs.py, go to line 139, you will see this
' ',
delete it, save the document.
See attached screenshot

**Comment by theruleof4** [comment]
Ok. I'll try and see the results

**Comment by theruleof4** [comment]
Hm... seems not worked, I removed it and looked like this:
```
def text_fixer(text, debug=False):
    replaceMe = ['\n', '“', '”', '、', '//', ';', '<',
                 '>', '-', '_', '=', '+', '*', '&', '^',
                 '%', '#','@', '$', '.', '.', '~', ' ', '\t']
```
But the words still with no space. 
Another thing: the progress percent it been saved too, above all text.. like this:

```
(95%)
Somethingtobelikethis

```

**Comment by hamsolo474** [comment]
It's this, remove this.

**Comment by theruleof4** [comment]
Yeah, I've already removed and saved, I ran the script again and the problem was not fixed...
